argo_rollouts/assets/dashboards/argo_rollouts_overview.json

```diff
@@ -0,0 +1,1589 @@
+{
+    "author_name": "Datadog",
+    "description": "This dashboard provides a high-level overview of your Argo Rollouts deployment to help monitor your system.",
+    "layout_type": "ordered",
+    "template_variables": [
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "host",
+            "prefix": "host"
+        },
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "argo_rollouts_name",
+            "prefix": "argo_rollouts_name"
+        },
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "argo_rollouts_namespace",
+            "prefix": "argo_rollouts_namespace"
+        }
+    ],
+    "title": "Argo Rollouts Overview",
+    "widgets": [
+        {
+            "definition": {
+                "banner_img": "/static/images/logos/argo-rollouts_large.svg",
+                "layout_type": "ordered",
+                "show_title": false,
+                "title": "About Airflow",
+                "title_align": "left",
+                "type": "group",
+                "widgets": [
+                    {
+                        "definition": {
+                            "background_color": "transparent",
+                            "content": "This dashboard provides a high-level overview of your Argo Rollouts deployment to help monitor your system.\n",
+                            "font_size": "14",
+                            "show_tick": false,
+                            "text_align": "left",
+                            "tick_edge": "left",
+                            "tick_pos": "50%",
+                            "type": "note"
+                        },
+                        "id": 8764466536847944,
+                        "layout": {
+                            "height": 2,
+                            "width": 3,
+                            "x": 0,
+                            "y": 0
+                        }
+                    },
+                    {
+                        "definition": {
+                            "background_color": "transparent",
+                            "content": "**Refer to the following resources for more information**\n\n- [Datadog Argo Rollouts documentation](https://docs.datadoghq.com/integrations/argo_rollouts/)\n\n- [Available Argo Rollouts metrics](https://docs.datadoghq.com/integrations/argo_rollouts/#metrics) \n",
```

Comment:
Are these docs live or pending? These links didn't work for me

Reply:
These are not live yet. I think we're waiting for the release of the integration before we enable the docs for it.